src/main/java/seedu/address/logic/commands/DeleteMemberTaskCommand.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TASK;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Member;
+import seedu.address.model.task.Task;
+
+/**
+ * Deletes a task from the Member in the address book.
+ */
+public class DeleteMemberTaskCommand extends Command {
+
+    public static final String COMMAND_WORD = "deleteTask";
+    public static final String COMMAND_ALIAS = "delt";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes a specified task from the specified member "
+            + "by the index number used in the displayed member list "
+            + "and the task index used in the displayed task list. \n"
+            + "Parameters: MEMBER_INDEX (must be a positive integer) "
+            + PREFIX_TASK + "TASK_INDEX";
+
+    public static final String MESSAGE_DELETE_TASK_SUCCESS = "Task %1$s deleted from member %2$s";
+    private final Index memberIndex;
+    private final Index taskIndex;
+    /**
+     * Creates an DeleteMemberTaskCommand to delete the specified {@code Task}
+     */
+    public DeleteMemberTaskCommand(Index memberIndex, Index taskIndex) {
+        requireNonNull(memberIndex);
+        requireNonNull(taskIndex);
+
+        this.memberIndex = memberIndex;
+        this.taskIndex = taskIndex;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Member> lastShownList = model.getFilteredMemberList();
+
+        if (memberIndex.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_MEMBER_DISPLAYED_INDEX);
+        }
+
+        Member memberToDeleteTaskFrom = lastShownList.get(memberIndex.getZeroBased());
+
+        if (taskIndex.getZeroBased() >= memberToDeleteTaskFrom.getTasks().size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_TASK_DISPLAYED_INDEX);
+        }
+
+        Task taskToDelete = memberToDeleteTaskFrom.getTasks().get(taskIndex.getZeroBased());
```

Comment:
Might need to make sure the list is sorted since it is displayed as a sorted list but it might not be stored in a sorted manner in the List<Task>